### PR TITLE
Enable to use custom helpers in error middleware, closes #438 

### LINKED ISSUE #438: Helper functions not available in rescue_from
Hi, we are trapping errors in rescue_from but unable to log them because the logger variable appears out of scope. Any ideas on how to access it? Thanks.

### PR DESCRIPTION
**Pull request by namusyaka**
Hello, everyone.
I'd love to use custom helpers in the Error middleware.
This change might be able to close #438 issue.
Could you review this?

Thanks.

### CODE REVIEW

File: spec/grape/api_spec.rb [hunk 1]
```diff
@@ -1292,6 +1292,20 @@ def three
       expect { get '/exception' }.to raise_error(RuntimeError, 'rain!')
     end
 
+    it 'should be able to use custom helpers defined by using #helpers method' do
```
Comment (dblock) [inline spec/grape/api_spec.rb]: should be able to use -> uses

**Comment by dblock** [comment]
This looks good to me. Update CHANGELOG please, squash commits. Maybe needs a README change, if you find any suitable place.

**Comment by namusyaka** [comment]
@dblock Thank you. Fixed and force pushed.

**Comment by dblock** [comment]
Merged via https://github.com/ruby-grape/grape/commit/951ae51cc0eccec6c5225c2aaf16f091829062dd (slightly amended text). Thanks!

**Pull request closed without merging**